install/install.sh

```diff
@@ -257,21 +257,43 @@ detect_target() {
   printf '%s' "${target}"
 }
 
-
 confirm() {
   if [ -z "${FORCE-}" ]; then
+    trap 'echo; error "Interrupted (please re-run with the '--yes' option)"; exit 130' INT
```

Comment:
nits: This trap setup is duplicated on L276. Consider using a helper function to avoid repetition.